src/plugins/proxy/src/plugin.cpp

```diff
@@ -299,15 +300,35 @@ ov::Any ov::proxy::Plugin::get_property(const std::string& name, const ov::AnyMa
     return get_core()->get_property(get_primary_device(device_id), name, {});
 }
 
+ov::SoPtr<ov::IRemoteContext> ov::proxy::Plugin::create_proxy_context(
+    const ov::SoPtr<ov::ICompiledModel>& compiled_model,
+    const ov::AnyMap& properties) const {
+    auto dev_name = get_device_name();
+    auto dev_idx = get_device_from_config(properties);
+    auto has_dev_idx = is_device_in_config(properties);
+    auto is_new_api = get_core()->is_new_api();
+    ov::SoPtr<ov::IRemoteContext> device_context;
+    ov::SoPtr<ov::IRemoteContext> remote_context;
+    try {
+        device_context = compiled_model->get_context();
+        if (!device_context._so)
+            device_context._so = compiled_model._so;
+        remote_context =
+            std::make_shared<ov::proxy::RemoteContext>(device_context, dev_name, dev_idx, has_dev_idx, is_new_api);
+    } catch (const ov::NotImplemented&) {
```

Comment:
is this exception enough? if underlying hardware still 1.0, I suppose still IE exceptions

Reply:
Yes it is ok, in case of legacy API it is ok if method will throw an exception.